diagnostic.md

```diff
@@ -20,6 +26,10 @@ c = c + d;
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d;` and explain what they do.
 
+the value of c is 2
+the operator '+' is the addition operator which adds c and d, so in this scenario 5 and 2
+the operator '=' sets the new value of c
+
```

Comment:
The value of c is in fact 7. Here's a breakdown line by line:
let c = 5 (c is equal to 5)
let d = 2 (d is equal to 2)
c = c + d (c is now assigned the value of c + d which equals 5 + 2, which equals 7)